gfx/sfml_painter.cpp

```diff
@@ -20,10 +20,20 @@
 namespace gfx {
 namespace {
 
+std::filesystem::recursive_directory_iterator get_font_dir_iterator(std::filesystem::path const &path) {
```

Comment:
Maybe use a function-try-block here and return directly? Makes it a bit more obvious what's going on imo.

```
auto get_font_dir_iterator(auto const &path) try {
  return iterator(path);
} catch (...) {
  return {};
}
```

Reply:
Yes, that looks nicer!